ImportError: No module named 'pandas.compat.numpy_compat' when running tests

Following the [getting started guide](http://taxcalc.readthedocs.org/en/latest/contributor_guide.html), upon running `py.test -m "not requires_pufcsv"` a user is presented with 13 errors, all:

```
...\Miniconda3\envs\taxcalc-dev\lib\site_packages\pandas\__init.py:22: in (module)
     from pandas.compat.numpy_compat import *
E   ImportError: No module named 'pandas.compat.numpy_compat'
```

`pandas.compat.numpy_compat.py` was added to pandas in January 2016, whereas the version of pandas specified in `environment.yml` was released in June 2015. Why/how a pandas release from 2015 is requiring code that wasn't added until 2016, I have no idea. Hopefully someone can improve my understanding of the package distribution on this point.

However, if I remove the version constraint in environment.yml, pandas 0.18.0 installs successfully and all the tests pass (besides the unrelated one mentioned in #660). Is there current reason for the `<=0.16.2` constraint?

--- a/environment.yml
+++ b/environment.yml
@@ -3,8 +3,8 @@ dependencies:
 - pytest
 - setuptools
 - python
-- numpy
-- pandas <=0.16.2
+- numpy =1.10.4
+- pandas =0.18.0
 - numba
 - toolz
 - six

